lib/metadata/linux/LinuxSystemd.rb

```diff
@@ -119,6 +129,10 @@ def service_targets_xml(service)
       end
     end
 
+    def duplicated_link?(file)
+      existing_dirs.any? { |dir| @fs.getLinkPath(file).start_with?(dir) } if @fs.fileSymLink?(file)
```

Comment:
I'm not sure `start_with?` is good enough here, unless the values in `dir` end in a `/`. For example: `"/etc/dir11".start_with?("/etc/dir1")` is true.

Reply:
Good catch, I replaced `start_with?` with array comparison.